OpenSim/Simulation/Model/Station.h

```diff
@@ -26,11 +26,11 @@
 
 // INCLUDE
 #include "OpenSim/Simulation/Model/Point.h"
+#include "PhysicalFrame.h"
 
```

Comment:
Any reason PhysicalFrame.h include is needed now?

Reply:
Not sure, why this was touched. I'll revert and see.

Reply:
Done.

Reply:
gcc and clang correctly flag forward declaration as insufficient to define the `Socket<PhysicalFrame>`. Not sure why visual studio lets it through.